Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -1429,13 +1450,19 @@ static float currentBacklightLevel()
 }
 #endif
 
-void WebProcess::unblockServicesRequiredByAccessibility(const Vector<SandboxExtension::Handle>& handleArray)
+void WebProcess::unblockServicesRequiredByAccessibility(Vector<SandboxExtension::Handle>&& handles)
 {
-#if PLATFORM(IOS_FAMILY)
-    bool consumed = SandboxExtension::consumePermanently(handleArray);
-    ASSERT_UNUSED(consumed, consumed);
-#endif
+    Vector<RefPtr<SandboxExtension>> extensions;
+    for (size_t i = 0; i < handles.size(); i++) {
+        auto extension = SandboxExtension::create(WTFMove(handles[i]));
+        extension->consume();
+        extensions.append(WTFMove(extension));
+    }
```

Comment:
This might be cleaner as `WTF::map()` where the lambda you provide takes a move argument:

```
Vector<RefPtr<SandboxExtensions>> extensions = WTF::move(handles, [](SandboxExtension::Handle&& handle) {
    ... etc. ...
```